### DISCUSSION TITLE
How does one update a time series regression model?

### DISCUSSION THREAD

**Discussion by ys1382** [timeseriesAI/tsai#38] [Q&A]
If I train a model on a lot of time series data, and periodically get the latest datapoint, how can I update my model without training from scratch?

**Comment by oguiza** [comment]
Hi @ys1382,
Once you train a model from scratch, you should handle it as a 'pre-trained' model. What I mean is that if you get new data, you can create a new TSDataLoaders (with the train set being the new data, and valid being any control data you want to use) and fine-tune it. This avoids the need to completely re-train the model every time you get new data.
This is the same principle you follow in transfer learning with vision models if you are familiar with it.
